Shifty.App/Components/UnusedTickets.razor

```diff
@@ -0,0 +1,95 @@
+@namespace Components
+@using System.ComponentModel.DataAnnotations
+@using Shifty.App.Services
+@using Shifty.Api.Generated.AnalogCoreV1
+@using Shifty.Api.Generated.AnalogCoreV2
+@using Shifty.App.DomainModels
+@using Shifty.App.Shared
+@using Shared
+@using LanguageExt.UnsafeValueAccess
+@inject IUnusedTicketsService _unusedTicketsService
+@inject ISnackbar Snackbar
+
+<MudContainer MaxWidth="MaxWidth.Medium" Style="margin-top: 20px;">
+    <MudDataGrid
+        T="UnusedTicket"
+        Items="@Items"
+        Height="500px"
+        FixedFooter>
+        <ToolBarContent>
+            <MudText Typo="Typo.h6">Unused Tickets</MudText>
+            <MudSpacer />
+            @if (_loading)
```

Comment:
We can discuss using the MudLoading from the extension library for our loading indicator

Reply:
I hate to be that guy but you kinda told me otherwise last time 😂
https://github.com/AnalogIO/shifty-webapp/pull/16#discussion_r1357270272 

I mean it would be easier code-wise to use MudLoading but it is not as configurable. I tried to use our own loading indicator as the `LoadingContent` attribute on the `MudDataGrid` but it only wanted to use the default MudBlazor loading indicator. It is possible to change only the color it seems.